receiver/awscontainerinsightreceiver/internal/host/nodeCapacity.go

```diff
@@ -0,0 +1,65 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package host
+
+import (
+	"os"
+
+	"github.com/shirou/gopsutil/cpu"
+	"github.com/shirou/gopsutil/mem"
+	"go.uber.org/zap"
+)
+
+const (
+	GoPSUtilProcDirEnv = "HOST_PROC"
+)
+
+type NodeCapacity struct {
```

Comment:
It's not really related to this file but do we consider using `/rootfs/proc/meminfo` to calculate (another) node metrics instead of relying on cgroup's working set (which is not that accurate as it considers file cache).

Reply:
I think we can consider to add them after the feature parity is done.